components/workspaces/staging/base/server/config/server/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- deployment.yaml
+- service.yaml
+- route.yaml
```

Comment:
Take this out as well.

```suggestion
```

Reply:
Got it

Reply:
since we need to bring back when we move to base up a level, should I just remove from customize file reference?

Reply:
Nope, we also have to do this for production environments, so let's just do it correctly here.  Any route we need to have lives in the environment folder underneath `staging/` or `production/`.